Kingsley_ansible_playbook.yml

```diff
@@ -0,0 +1,94 @@
+---
+- name: Configure TodoApp Environment
+  hosts: webservers
+  become: yes
+  remote_user: ec2-user
+ 
+  vars_prompt:
+    - name: trello_api_key
+      prompt: "Enter Trello API Key"
+      private: yes
+      
+    - name: trello_api_token
+      prompt: "Enter Trello API Token"
+      private: yes
+
+    - name: board_id
+      prompt: "Enter Trello Board ID"
+      private: no
+      
+    - name: to_do_list_id
+      prompt: "Enter To Do List ID"
+      private: no
+
+    - name: done_list_id
+      prompt: "Enter Done List ID"
+      private: no
```

Comment:
You ask for these three variables, but since you have hardcoded the values in your `.env.j2` file, they won't actually be used, so we might as well get rid of them!
```suggestion
```